crates/matrix-sdk/src/send_queue.rs

```diff
@@ -556,6 +556,20 @@ impl RoomSendQueue {
             self.inner.notifier.notify_one();
         }
     }
+
+    /// Unwedge a local echo identified by its transaction identifier and try to
+    /// resend it.
+    pub async fn unwedge(&self, transaction_id: &TransactionId) -> Result<(), RoomSendQueueError> {
+        self.inner
+            .queue
+            .mark_as_unwedged(transaction_id)
+            .await
+            .map_err(RoomSendQueueError::StorageError)?;
+
+        self.inner.notifier.notify_one();
```

Comment:
Can you add a comment here similar to others doing the same thing, explaining why we're notifying?